walletlibrary/src/main/java/com/microsoft/walletlibrary/VerifiedIdClientBuilder.kt

```diff
@@ -62,6 +65,13 @@ class VerifiedIdClientBuilder(private val context: Context) {
         logger.addConsumer(logConsumer)
     }
 
+    fun with(extension: VerifiedIdExtension): VerifiedIdClientBuilder {
+        // TODO: Add prefer headers to RequestResolverFactory
+        preferHeaders.addAll(extension.prefer)
+        // TODO: lookup RequestProcessors by extension associated types and inject extensions
```

Comment:
I believe there is code here in the internal branch; likely the branch I had to set up the extension does not have it yet. Will need to pull in more commits before publishing this PR.